api/container/containerv2/workers.go

```diff
@@ -91,7 +91,7 @@ type VolumeRequest struct {
 type Workers interface {
 	ListByWorkerPool(clusterIDOrName, workerPoolIDOrName string, showDeleted bool, target ClusterTargetHeader) ([]Worker, error)
 	ListWorkers(clusterIDOrName string, showDeleted bool, target ClusterTargetHeader) ([]Worker, error)
-	ListClassicWorkers(clusterIDOrName string, showDeleted bool, target ClusterTargetHeader) ([]Worker, error)
```

Comment:
Removing this method may impact other's users. We have some standalone users who uses this SDK

Reply:
Can we mark it as deprecated and add new method to support (ListAllWorkers)

Reply:
This method was added [two months](https://github.com/IBM-Cloud/bluemix-go/pull/448/files#diff-fd51d6ddfb85b1d45ffe6e1b203e0e19e45709fa5f60c606458081a013d774a8R94) ago and it's not working under normal circumstances (regional api is not targeted), If you want, we can keep the bad version as well, I just saw no point currently as it's causing issues and is most likely only used by us in terraform (as the change is recent)

Given this information, do you still want to keep it around?